Stratis.Bitcoin/Features/BlockStore/BlockStoreLoop.cs

```diff
@@ -14,6 +14,11 @@
 
 namespace Stratis.Bitcoin.Features.BlockStore
 {
+    /// <summary>
+    /// The BlockStoreLoop simultaneously finds and downloads blocks
+    /// and stores them in the BlockRepository
+    /// <see cref="DownloadAndStoreBlocks"/>
+    /// </summary>
     public sealed class BlockStoreLoop
     {
         internal readonly ConcurrentChain Chain;
```

Comment:
XMLDOC rule - missing comment here, something as simple as `/// <summary>Best chain of block headers.</summary>` is good enough